clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/model/EnableDisableDestroyAzureServerGroupDescription.groovy

```diff
@@ -18,5 +18,8 @@ package com.netflix.spinnaker.clouddriver.azure.resources.servergroup.model
 
 class EnableDisableDestroyAzureServerGroupDescription extends AzureServerGroupDescription{
   String serverGroupName
+  String region
```

Comment:
region is already defined in the base class AzureResourceOpsDescription (all our descriptions are expected to extend that class)